### PR TITLE
Fe skeleton

### PR DESCRIPTION
**Pull request by berri-puff**

### CODE REVIEW

**Comment by ChrisMaylor92** [comment]
Looks good to me!

File: src/App.css
Comment (aislingkelly) [inline src/App.css:1]: Looks good!

**Comment by addleton** [comment]
I dont agree with Chris, please use comic sans font

File: src/components/MeetPage/MeetingMap.tsx
Comment (aislingkelly) [inline src/components/MeetPage/MeetingMap.tsx:1]: Also looks good!

**Review by aislingkelly — COMMENTED** [review]
Finishing this review :)

**Review by ChrisMaylor92 — COMMENTED** [review]
Happy!

**Pull request merged**